### PR TITLE
Feature/integrations/klaviyo

### PR DESCRIPTION
**Pull request by Aduralere54256703**
klaviyo help center modification

### CODE REVIEW

File: integrations/klaviyo.mdx [hunk 1]
```diff
@@ -1,8 +1,32 @@
 ---
 title: 'Klaviyo'
-description: 'Text, title, and styling in standard markdown'
+description: 'Klaviyo help center article'
 icon: 'text-size'
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas et eros iaculis tortor dapibus cursus.
-Curabitur quis sapien nec tortor dictum gravida.
+Klaviyo is an email and sms automation platform that helps merchants build advanced marketing flows.
```
Comment (Aduralere54256703) [inline integrations/klaviyo.mdx:7]: Adding text to the klaviyo help center document

**Review by Aduralere54256703 — COMMENTED** [review]
klaviyo help center documentation

**Review by kele-lydia — COMMENTED** [review]
Check comment

**Review by kele-lydia — CHANGES_REQUESTED** [review]
Markdown not properly displayed
Screenshot of Klaviyo - Starter Kit

**Pull request closed without merging**